security/pkg/server/ca/authenticate/authenticator.go

```diff
@@ -101,7 +102,7 @@ func NewIDTokenAuthenticator(aud string) (*IDTokenAuthenticator, error) {
 		return nil, err
 	}
 
-	verifier := provider.Verifier(&oidc.Config{ClientID: aud})
+	verifier := provider.Verifier(&oidc.Config{ClientID: aud, Now: func() time.Time { return time.Now().Add(-30 * time.Second) }})
```

Comment:
How is this related to https://github.com/istio/istio/issues/22750?

Reply:
It's not. The actual code is in https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/jwt_authn/authenticato.cc, line 240:
const Status status = ::google::jwt_verify::verifyJwt(*jwt_, *jwks_data_->getJwksObj());.

Closing this.